packages/dnb-eufemia/src/components/date-picker/DatePickerCalc.ts

```diff
@@ -251,7 +251,7 @@ export function convertStringToDate(
   date: string | Date,
   { date_format = null }: { date_format?: string | null } = {}
 ): Date {
-  if (date === null) {
+  if (date === null || date === undefined) {
```

Comment:
```suggestion
  if (!date) {
```

Can we simply go for this?

Reply:
Si